packages/apfelxx/README.md

```diff
@@ -0,0 +1,31 @@
+# APFEL++: A PDF evolution library in C++
+
+## Dependencies
+
+- `pybind11`
+
+## Installation
+
+```shell
+cd apfelxx
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=/your/installation/path/ ..
+make && make install
+```
+
```

Comment:
I would also add a comment here (for convenience) on what explicitly should be done if the installation path is not the same as the default.
```shell
export PATH=/your/installation/path/bin:$PATH
export LD_LIBRARY_PATH=/your/installation/path/lib:$LD_LIBRARY_PATH
```
It can be easy for new users to overlook this sometimes.

Reply:
If you wish, this bit we can in a single README inside `/packages/`, since it's the same for all of them :)

Reply:
> If you wish, this bit we can add in a single place inside `/packages/README.md`, since it's the same for all of them :)

Yes, this would be best indeed!